nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Sp maybe we can get something done now

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-maybe-we-can-get-something-done-now`
4. check list of computers, add a computer, view the details of a computer, and try to delete a computer (check that you can only delete a computer that is unassigned)

# Checklist:
